src/pages/Account/Account.scss

```diff
@@ -121,6 +121,7 @@ $page-max-width: 1240px;
   justify-content: center;
   flex: 1;
   background-color: #1a1d24;
+  height: 100%;
```

Comment:
Is `100%` fine here? Should it be `100vh` so that it is always 100% of the viewport (browser) height, rather than relying on the size of it's nearest parent?

Reply:
100% results in taking 100% of the viewport

Reply:
Not technically. 100% is 100% relevant to its parent container. `100vh` always represents 100% of the viewport.

- https://www.lullabot.com/articles/unexpected-power-of-viewport-units-in-css

Reply:
Technically, you are correct, but practically, 100%, in this case, results in 100% viewport. I don't think we have time for this discussion, so changed in 
https://github.com/sparkletown/sparkle/pull/1237/commits/33d029f0825d052343fabc5be8c570e31b3bcabe

Reply:
I believe it's only practically that way because of the 100% you set on the body/etc, which I believe is incorrect.

Thankyou for changing it.